ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -365,7 +385,7 @@ calcParserModules ghcFlavor = do
   copyFile mainFile rootModulePath
   lib <- mapM readCabalFile (cabalFileLibraries ghcFlavor)
   cabalPackageDb <- cabalPackageDb =<< ghcNumericVersion
-  let includeDirs = map ("-I" ++) (ghcLibParserIncludeDirs ghcFlavor)
+  let includeDirs = map ("-I" ++) (ghcLibParserIncludeDirs True ghcFlavor)
```

Comment:
Since ghcLibParserIncludeDirs ignores its first argument, we can drop this `True`

Reply:
here we pass through the value of `forDepends` from a context in which it is true. let us continue to do that though it's currently not strictly required.